apps/form-builder/src/app/api/forms/[formId]/formSubmissions/route.ts

```diff
@@ -0,0 +1,12 @@
+// function to get all form submisions
+import { getFormSubmissions } from "@/server/actions/formSubmissions";
+import { NextRequest, NextResponse } from "next/server";
+import { zObjectId } from '@hack4impact-utk/internal-models';
+
+export async function GET (_request: NextRequest, { params }: { params: { formId: string } }) {
+    // if formId is valid, get form submissions and return them with status code 200
+    if(zObjectId.safeParse(params.formId).success) {
+        const formSubmissions = getFormSubmissions(params.formId)
```

Comment:
change this line to `const formSubmissions = getFormSubmissions(params.formId)`. If you hover over the `getFormSubmissions` function, you can see it returns something of type `Promise<FormSubmission[] | null>`. This says "in the future, I promise to return an array of form submissions, or return null"

However, we don't want the form submissions in the future, we want to sit and wait until we have them. Hence adding the word `await`. Anytime you see the word `Promise`, or the word `async` in a function you are calling, it's a good indicator that you need to add the word `await`